.github/workflows/workflow.yml

```
@@ -0,0 +1,39 @@
+name: Build & public container image
+on:
+  push:
+    tags:
+      - v**
+
+env:
+  REGISTRY: docker.io
+  IMAGE_ORG: ${{ secrets.DOCKERHUB_USERNAME }}
+  IMAGE_NAME: klaytn-load-tester
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: ${{ github.event_name != 'pull_request' }}
```

Why do we need this expression here?
```suggestion
          push: true
```